astropy/samp/tests/test_hub_script.py

```diff
@@ -21,6 +21,7 @@ def teardown_function(function):
 
 @pytest.mark.slow
 def test_hub_script():
-    sys.argv.append("-m")  # run in multiple mode
-    sys.argv.append("-w")  # disable web profile
+    sys.argv.append("-m")       # run in multiple mode
+    sys.argv.append("-w")       # disable web profile
```

Comment:
Although minor, while we're at it: aligning comments is not worth the cost (making `git blame` less useful), so I suggest reverting this bit
```suggestion
    sys.argv.append("-m")  # run in multiple mode
    sys.argv.append("-w")  # disable web profile
```